atlas/src/shared/components/VideoPreview/VideoPreviewBase.styles.tsx

```diff
@@ -31,16 +31,33 @@ export const CoverWrapper = styled.div`
   width: 100%;
 `
 
-export const CoverContainer = styled.div`
+export const CoverContainer = styled.div<ClickableProps>`
   position: relative;
   width: 100%;
   height: 0;
   padding-top: 56.25%;
-
   transition-property: box-shadow, transform;
   transition-duration: 0.4s;
   transition-timing-function: cubic-bezier(0.165, 0.84, 0.44, 1);
   animation: ${fadeIn} 0.5s ease-in;
+  cursor: ${(props) => (props.clickable ? 'pointer' : 'auto')};
+  :hover {
+    ${(props) =>
+      props.clickable
```

Comment:
I found it a good practice to separate conditional styles to a separate `css` constant. That way it's much easier to read the styles. I know this is copied but perhaps we could do that anyway?

Reply:
Good point, will change that